tools/env/index.ts

```diff
@@ -0,0 +1,20 @@
+import path from 'path'
+
+import dotenv from 'dotenv'
+
+const env = process.env.NODE_ENV || 'development'
+if (env !== 'production') {
+  // Don't write production .env files!
+
+  const url = path.join(__dirname, `./env/${env}.env`)
+  const result = dotenv.config({
+    path: url,
+  })
+
+  console.log('RESULT: ', result)
+  console.log('WAT: ', process.env.DATABASE_URL)
+
+  if (result.error) {
+    throw result.error
+  }
+}
```

Comment:
Promise is last I say this. But take a look at the code. Is super well done
https://github.com/t3-oss/t3-env/blob/main/packages/core/src/index.ts#L215

And small. I would use it in this package as starting point

Reply:
what for? this package does not have type validation of env vars (it's implemented in the other packages)